### ISSUE TITLE
[Enhancement]: List existing **and** new chapter info side-by-side in Edit > Edit Chapters > Lookup 

### ISSUE THREAD

**Issue by iconoclasthero** [advplyr/audiobookshelf#2640]
### Describe the feature/enhancement

![Screenshot from 2024-02-21 09-27-42](https://github.com/advplyr/audiobookshelf/assets/535962/ca77e0d6-8c6b-4616-86e3-ba1c64255359)

I can see the new information, but why not list the existing chapter information there was well?  In this case, both have 20 chapters but they are hours off by chapter 14.  Since it is displayed in a much bigger format and grayed out, the existing information displayed is much less useful than it could be.

![Screenshot from 2024-02-21 09-59-49](https://github.com/advplyr/audiobookshelf/assets/535962/c5a0d546-b044-4540-b080-a8597b07299f)

**Comment by iconoclasthero** [comment]
Looking at mediainfo in the terminal and comparing to the screenshot above I was able to figure out what was going on, but it would be **much** easier if the two sets of chapter information were displayed side by side for comparison.

**Comment by hobesman** [comment]
Yeah in that particular case it looks like the existing chapters start at Chapter 1 and the new chapters start with "Intro" as 1 and and "Chapter 1" as 2, and so on. But that's besides the point of your request.

**Comment by iconoclasthero** [comment]
Yeah, in this case, seeing them side by side made that very clear.

**Comment by nichwall** [comment]
I tried to come up with a better chapter editor page if you'd like to help improve it. I made it into a new discussion so it's easier to combine the chapter editor page discussions. https://github.com/advplyr/audiobookshelf/discussions/3011
